strax/run_selection.py

```diff
@@ -283,6 +283,66 @@ def define_run(self: strax.Context,
                            " run definition")
 
 
+@strax.Context.add_method
+def available_for_run(self: strax.Context,
+                      run_id: str,
+                      include_targets: ty.Union[None, list, tuple, str] = None,
+                      exclude_targets: ty.Union[None, list, tuple, str] = None,
+                      pattern_type: str = 'fnmatch') -> pd.DataFrame:
```

Comment:
I also have to start using this, I am not at all used to this yet.

Reply:
It's super useful, especially since your PyCharm also uses this type checking